tardis/model/base.py

```diff
@@ -595,7 +595,6 @@ def from_csvy(cls, config, atom_data=None):
             )
             density_0 = csvy_model_data["density"].values * density_unit
             density_0 = density_0.to("g/cm^3")[1:]
```

Comment:
Probably should get rid of the `[1:]` at the end line 597, just to make sure the dimension is consistent.